backend/api/views.py

```diff
@@ -357,6 +358,7 @@ def metric_by_id(request, exp_id, metric_id):
 
 
 @api_view(['POST'])
+@permission_classes((AllowAny,))
```

Comment:
Is this safe? I imagine it's coming locally so should be fine

Reply:
The streaming API doesn't send any auth tokens so this has to be there.